docs/content/development/extensions-core/test-stats.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: doc_page
+---
+
+# Test Stats Aggregators
+
+Incorporates test statistics related aggregators, including z-score and p-value. Please refer to [https://www.paypal-engineering.com/2017/06/29/democratizing-experimentation-data-for-product-innovations/](https://www.paypal-engineering.com/2017/06/29/democratizing-experimentation-data-for-product-innovations/) for background and details.
+
+Make sure to include `druid-stats` extension in order to use these aggregrators.
+
+## Z-Score for two sample ztests post aggregator
+
+Please refer to [https://www.isixsigma.com/tools-templates/hypothesis-testing/making-sense-two-proportions-test/](https://www.isixsigma.com/tools-templates/hypothesis-testing/making-sense-two-proportions-test/) and [http://www.ucs.louisiana.edu/~jcb0773/Berry_statbook/Berry_statbook_chpt6.pdf](http://www.ucs.louisiana.edu/~jcb0773/Berry_statbook/Berry_statbook_chpt6.pdf) for more details.
+
+z = (p1 - p2) / S.E.  (assuming null hypothesis is true)
+
+where S.E. stands for standard error, and
+
+S.E. = sqrt{ p1 * ( 1 - p1 )/n1 + p2 * (1 - p2)/n2) }
+
+(p1 – p2) is the observed difference between two sample proportions.
+
+### zscore2sample post aggregator
+* **`zscore2sample`**: calculate the z-score using two-sample z-test while converting binary variables (***e.g.*** success or not) to continuous variables (***e.g.*** conversion rate).
+
+```json
+{
+  "type": "zscore2sample",
+  "name": "<output_name>",
+  "fields": [<count 1 (post_aggregator1)>, <sample size 1 (post_aggregator2)>, <count 2 (post_aggregator3)>, <sample size 2 (post_aggregator4)>]
```

Comment:
It looks that the number of arguments is always 4 and their order matters. If so, I think it would be better to receive 4 individual arguments instead of a fields list because it is less error-prone and intuitive which arguments are needed. What do you think?